pyhomematic/_devices.py

```diff
@@ -613,6 +631,11 @@ def on(self):
     def off(self):
         """Turn light off."""
         self.level = 0.0
+
+    @property
+    def working(self):
+        """Return True of False if working or not"""
+        return self._working
```

Comment:
Same as for the rollershutter. If the WORKING-state is None and can be fetched via proxy, it should be done. We always want some value that corresponds to the real state. None seems like unknown. But the state is known if we ask the CCU.